docs/measure/index.md

```diff
@@ -114,6 +114,88 @@ if __name__ == "__main__":
         avg_energy = sum(map(lambda m: m.total_energy, steps)) / len(steps)
         print(f"One step takes {avg_time} s and {avg_energy} J for the CPU.")
 ```
+## Metric Monitoring
```

Comment:
We need to have a section on our assumptions on Prometheus. For instance, we assume that the Prometheus Push Gateway is deployed, so we need to talk about that and have a brief section for how to do that or a link to an official documentation. Are there other assumptions?